target_chains/ethereum/contracts/forge-test/Pyth.t.sol

```diff
@@ -309,4 +391,226 @@ contract PythTest is Test, WormholeTestUtils, PythTestUtils {
             MAX_UINT64
         );
     }
+
+    function testParseTwapPriceFeedUpdates() public {
```

Comment:
the tests look good, but i don't like the idea that you create the price feed message and then somehow convert them to twap by calling `createBatchedTwapUpdateDataFromMessages` which includes some math. I recommend you manually set TwapMessage (or CumulativeMessage) in your update and then check the result without doing math on the code.

Reply:
well i actually like you to do it for all the feeds. to avoid redundancy you can store them in a variable in test setUp and just change the parts needed (like changing timestamps/..)